src/main/java/pro/sky/homework_2_9/DepartmentServiceImpl.java

```diff
@@ -2,25 +2,24 @@
 
 import org.springframework.stereotype.Service;
 
-import java.util.*;
-import javax.swing.*;
+import java.util.Collection;
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
 import java.util.stream.Collectors;
 
 @Service
-public class DepartmentServiceC implements DepartmentServiceI{
+public class DepartmentServiceImpl implements DepartmentService{
 
-    private final EmployeeServiceC employeeService;
+    private final EmployeeServiceImpl employeeService;
```

Comment:
```suggestion
    private final EmployeeService employeeService;
```
инъекцию всегда надо делать по интерфейсу, а не реализации